go-controller/pkg/ovn/default_network_controller.go

```diff
@@ -770,12 +773,24 @@ func (h *defaultNetworkControllerEventHandler) AddResource(obj interface{}, from
 			if err = h.oc.addUpdateLocalNodeEvent(node, nodeParams); err != nil {
 				klog.Infof("Node add failed for %s, will try again later: %v",
 					node.Name, err)
-				return err
+				aggregatedErrors = append(aggregatedErrors, err)
+			}
+			if err = h.oc.addIPToHostNetworkNamespaceAddrsSet(node); err != nil {
```

Comment:
so another thing to think about:
```
if err = h.oc.addIPToHostNetworkNamespaceAddrsSet(node); err != nil {
				klog.Errorf("Failed to add IPs to %s namespaced address_set: %v", config.Kubernetes.HostNetworkNamespace, err)
				h.oc.syncHostNetAddrSetFailed.Store(node.Name, true)
				aggregatedErrors = append(aggregatedErrors, err)
			}
```
is duplicated in the if, else, is there something preventing you from just moving this block to the very end and doing it once after the whole if,else? 
like really what's the point of putting this inside if and else if the if condition doesn't really matter?

Reply:
same for updateResource as well

Reply:
Agreed, silly mistake. Will rectify and push after testing it.